common/network-common/src/main/java/org/apache/spark/network/crypto/AuthEngine.java

```diff
@@ -224,7 +236,7 @@ private TransportCipher generateTransportCipher(
 
   private byte[] getTranscript(AuthMessage... encryptedPublicKeys) {
     ByteBuf transcript = Unpooled.buffer(
-        Arrays.stream(encryptedPublicKeys).mapToInt(k -> k.encodedLength()).sum());
+        Arrays.stream(encryptedPublicKeys).mapToInt(AuthMessage::encodedLength).sum());
```

Comment:
Let us minimize optional changes in this PR (this by itself is fine - but in general, we should reduce the scope of change), given we want to backport to older versions and want to minimize risk.

Reply:
+1 for @mridulm 's comment. This should be reverted.

Reply:
Reverted. Good call.